x-pack/legacy/plugins/apm/scripts/optimize-tsconfig/unoptimize.js

```diff
@@ -5,32 +5,21 @@
  */
 /* eslint-disable import/no-extraneous-dependencies */
 
-const path = require('path');
 const execa = require('execa');
-const fs = require('fs');
-const promisify = require('util').promisify;
-const removeFile = promisify(fs.unlink);
-const exists = promisify(fs.exists);
 
-const { apmRoot, filesToIgnore } = require('./paths');
+const { filesToIgnore } = require('./paths');
 
 async function unoptimizeTsConfig() {
   for (const filename of filesToIgnore) {
     await execa('git', ['update-index', '--no-skip-worktree', filename]);
     await execa('git', ['checkout', filename]);
   }
-
-  const apmTsConfig = path.join(apmRoot, 'tsconfig.json');
-  if (await exists(apmTsConfig)) {
-    await removeFile(apmTsConfig);
-  }
 }
 
 module.exports = {
-  unoptimizeTsConfig: () => {
-    return unoptimizeTsConfig().then(() => {
-      // eslint-disable-next-line no-console
-      console.log('Removed APM TypeScript optimizations');
-    });
+  unoptimizeTsConfig: async () => {
+    await unoptimizeTsConfig();
+    // eslint-disable-next-line no-console
+    console.log('Removed APM TypeScript optimizations');
```

Comment:
This should go to stderr too.

Reply:
It's not an error though, am I misunderstanding what you mean?

Reply:
`console.error` logs to stderr instead of stdout. stdout is meant for output that can be piped around, and stderr for messages like this. 

https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout

it's probably a controversial point though against "console.error is for errors" so either way I guess.

Reply:
I got that, but you might have commented in the wrong file? Did you meant to comment on `scripts/unoptimize-tsconfig.js`? (I've changed it to console.error there as well).